core/src/main/java/org/frankframework/pipes/ReplacerPipe.java

```diff
@@ -62,38 +62,34 @@ public void configure() throws ConfigurationException {
 				replace = replace != null ? replace.replace(lineSeparatorSymbol, System.getProperty("line.separator")) : null;
 			}
 		}
-		if (isReplaceNonXmlChars()) {
-			if (getReplaceNonXmlChar() != null) {
-				if (getReplaceNonXmlChar().length() > 1) {
-					throw new ConfigurationException("replaceNonXmlChar [" + getReplaceNonXmlChar() + "] has to be one character");
-				}
-			}
+
+		if (isReplaceNonXmlChars() && getNonXmlReplacementCharacter() != null && getNonXmlReplacementCharacter().length() > 1) {
```

Comment:
De volgende keer als je dit ziet kan je ook
```suggestion
		if (isReplaceNonXmlChars() && StringUtils.isNotEmpty(getNonXmlReplacementCharacter()) {
```
gebruiken.